idd/Energy+.idd.in

```diff
@@ -1287,19 +1287,23 @@ RunPeriod,
        \minimum 1
        \maximum 31
        \type integer
-  N3 , \field End Month
+  N3,  \field Begin Year
+       \note start year of the simulation, if this field is specified it must agree with the Day of Week for Start Day
+  N4 , \field End Month
        \required-field
        \minimum 1
        \maximum 12
        \type integer
-  N4 , \field End Day of Month
+  N5 , \field End Day of Month
        \required-field
        \minimum 1
        \maximum 31
        \type integer
+  N6,  \field End Year
+       \note end year of simulation, if specified
   A2 , \field Day of Week for Start Day
-       \note  =<blank - use WeatherFile>|Sunday|Monday|Tuesday|Wednesday|Thursday|Friday|Saturday];
-       \default UseWeatherFile
```

Comment:
@mjwitte  Right now it will compute the correct weekday when this is left blank and if not it throws a severe error if the begin year and weekday don't match. I could change that if you feel strongly about it.

Reply:
That works, but then it shouldn't have a hard default in the IDD and the note should be updated to say that.

Reply:
@mjwitte I guess it's really a soft default, so I agree that should be changed. Will do that once I get this merge thing worked out.